hermes/brokers/hopskotch.py

```diff
@@ -384,16 +397,14 @@ def _get_hop_topic_pk(topic_name, user_api_token) -> int:
     return hop_topic_pk
 
 
-def _get_hop_credential_pk(username, credential_name, user_pk=None, user_api_token=None):
+def _get_hop_credential_pk(username, credential_name, user_api_token: str, user_pk: int=None):
```

Comment:
For consistency, you might want to switch the order of the arguments in line 238.